CIP-0010/registry.json

```
@@ -55,6 +55,10 @@
     "transaction_metadatum_label": 1888,
     "description": "amphitheatre by the ape society"
   },
+  {
+    "transaction_metadatum_label": 1904,
+    "description": "Proof of origin and quality related verification data in supply chain"
```

```suggestion
    "description": "Proof of origin and quality verification data in supply chain"
```
Considering space is at a premium here, and that this is our longest `description`, wouldn't it mean the same thing without `related`?

When you have your new CIP with the full spec, we would be adding that here & therefore wouldn't need so many words beyond the CIP number: so this isn't vital & there'd be no point shortening it further.

---

On second thought, that introduces an ambiguity implying "proof of quality" (generally impossible) so better to leave it the way it is & trust it will be elaborated properly in the forthcoming CIP spec.